### ISSUE TITLE
eldev running into issues and causing build failures

### ISSUE THREAD

**Issue by Trevoke** [doublep/eldev#52]
Hello! I'm using eldev both locally and in my Github Actions to guarantee that my package remains green. For a while now, though, the build has been failing, and I have no idea why. I could use your help in troubleshooting this.

https://github.com/Trevoke/org-gtd.el/runs/3927553474
https://github.com/Trevoke/org-gtd.el/runs/3927553546
https://github.com/Trevoke/org-gtd.el/runs/3927553615

(related: https://github.com/Trevoke/org-gtd.el/issues/66 )

**Comment by doublep** [comment]
This is what I see in the logs (I checked only the latest failure):

    [00:15.712]  [2/7] Installing package ‘org-edna’ (1.1.2) from ‘gnu’...
    [00:15.712]  File ‘https://elpa.gnu.org/packages/org-edna-1.1.2.tar’ is not cached, retrieving...
    [00:15.871]  https://elpa.gnu.org/packages/org-edna-1.1.2.tar: Bad Request
    [00:15.871]  When accessing package archive ‘gnu’
    ...
    [00:15.871]  Finished erroneously on Mon Oct 18 14:02:39 2021

This appears to be a problem with GNU ELPA, and apparently it is already gone, I successfully ran `$ eldev clean all; eldev test` locally, which installed `org-edna` without problems.

In other words, I guess if you rerun the workflow, it will work now.

Maybe Eldev should include an option to auto-retry after certain errors with a delay, to be used on CI servers.

By the way, logs on GitHub are mixed up when both stderr and stdout are written to. I tried reporting this (https://github.com/isaacs/github/issues/1981), but no success.

**Comment by Trevoke** [comment]
Wow, good find.

I think you might want to just commit it with the version check, it sounds like a pretty significant bug that impacts whoever is using eldev with org as a dependency... And you can reach out to emacs-devel afterwards with no rush and no pressure?

**Comment by Trevoke** [comment]
My build is green! Finally! You're my hero, thank you!
